first-digit/first-digit.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	fmt.Println("The program prints the first digit of the given integer.")
+	fmt.Println("Enter the number:") 
+	var b int //0; 2786; -98; -1
+	fmt.Scanln(&b)
+	var c int
```

Comment:
style: It's better to declare variable `c` as the short assignment inside `for`, i.e. `for c := b; ...` and remove `var c int`.